### PR TITLE
Feat/next js firebase

### PR DESCRIPTION
**Pull request by himanshuchawla009**

### CODE REVIEW

File: examples/next-app-custom/pages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          <a target="_blank" href="http://web3auth.io/" rel="noreferrer noopener">
+            Web3Auth
+          </a>{" "}
+          & NextJS Example
+        </h1>
+        <Setting setNetwork={setWeb3AuthNetwork} setChain={setChain} />
+        <Main />
+      </Web3AuthProvider>
+      <footer className={styles.footer}>
+        <a href="https://github.com/Web3Auth/Web3Auth/tree/master/examples/next-app" target="_blank" rel="noopener noreferrer">
```
Comment (tainguyentt) [inline examples/next-app-custom/pages/index.tsx:28]:
```suggestion
        <a href="https://github.com/Web3Auth/Web3Auth/tree/master/examples/next-app-custom" target="_blank" rel="noopener noreferrer">
```

**Comment by tainguyentt** [comment]
I got blank screen when redirecting to openlogin
image

**Review by tainguyentt — COMMENTED** [review]
We should name examples consistently. We had `firebase-react-example`, should we name this `firebase-next-example`?
image

**Comment by himanshuchawla009** [comment]
> I got blank screen when redirecting to openlogin image

plz attach console logs if its happening every time.

**Review by tainguyentt — APPROVED** [review]
I have tested it, and it works. Thanks

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**